unit-tests/utilities/time/test-timer.cpp

```diff
@@ -31,20 +34,25 @@ TEST_CASE( "timer", "[Timer]" )
     t.start();
     CHECK_FALSE(t.has_expired());
 
-    std::this_thread::sleep_for(std::chrono::milliseconds(500));
+    std::this_thread::sleep_for(TEST_DELTA_TIME - std::chrono::milliseconds(400));
```

Comment:
May not work if TEST_DELTA_TIME is < 400ms

Reply:
Replaced with TEST_DELTA_TIME / 2